### DISCUSSION TITLE
Picow wifi module compiler error

### DISCUSSION THREAD

**Discussion by k-madsen** [earlephilhower/arduino-pico#877] [General]
Just trying to compile BasicHTTPSClient.ino I got this error 
WiFiClass.cpp:68: undefined reference to `_ZN9WiFiClass5beginEPKcS1_'
WiFiMulti.cpp:98: undefined reference to `_ZN9WiFiClass5beginEPKcS1_'

????
Regards
Kim Madsen

**Comment by k-madsenDK** [comment]
Remember to shift from pico to picow in the bordmaneger

**Reply by earlephilhower** [reply]
Yes, the `WiFi::begin()` method is not built unless you target the Pico W.  On platform.io this is `board = rpipicow`, or in the IDE just use the menu.
